docs/charon/charon_cli_reference.md

```diff
@@ -100,16 +103,20 @@ Usage:
 Flags:
       --clean                          Delete the cluster directory before generating it.
       --cluster-dir string             The target folder to create the cluster in. (default ".charon/cluster")
+      --definition-file string         Optional path to a cluster definition file or an HTTP URL. This overrides all other configuration flags.
       --fee-recipient-address string   Optional Ethereum address of the fee recipient
   -h, --help                           Help for cluster
-      --name string                    Optional cosmetic cluster name
+      --insecure-keys                  Generates insecure keystore files. This should never be used. It is not supported on mainnet.
+      --keymanager-addresses strings   Comma separated list of keymanager URLs to push validator key shares to. Note that multiple addresses are required, one for each node in the cluster, with node0's keyshares being pushed to the first address, node1's keyshares to the second, and so on.
+      --name string                    The cluster name
       --network string                 Ethereum network to create validators for. Options: mainnet, gnosis, goerli, kiln, ropsten, sepolia. (default "goerli")
-  -n, --nodes int                      The number of charon nodes in the cluster. Minimum is 4. (default 4)
+      --nodes int                      The number of charon nodes in the cluster. Minimum is 4. (default 4)
```

Comment:
Might have to be cautious we didn't break any old guides here if the -n flag is removed. Shouldn't be any old make files around any more, but maybe some sh scripts might have used the -n flag.

Reply:
We have removed makefiles in the cdvc repo as well as in places wherever possible. Haven't encountered any failing tests or builds, checked both cdvc and cdvn repos, charon tests pass, so shouldn't be an issue